src/migrations/1621511823100_token_metadata_queue.ts

```diff
@@ -28,6 +28,11 @@ export async function up(pgm: MigrationBuilder): Promise<void> {
     processed: {
       type: 'boolean',
       notNull: true,
+    },
+    retry_count: {
+      type: 'integer',
+      notNull: true,
+      default: 0,
```

Comment:
If this is moved to a new migration file (e.g. `alter table .. add column ...`) we can merge and deploy to beta testing without event-replay.